polkadot.json

```diff
@@ -7,68 +7,68 @@
         "id": 1,
         "imageUrl": "https://i.imgur.com/vF7XfK8.png",
         "title": "Meet the next-generation blockchain",
-        "details": "Welcome to the world of Polkadot, a blockchain platform that enables communication between different blockchains - created by Gavin Wood, one of the co-founders of Ethereum. It’s able to connect with different blockchains to form an interoperable, decentralized web."
+        "details": "Welcome to the world of Polkadot, a platform for blockchains and Web3 applications - created by Gavin Wood, one of the co-founders of Ethereum. It is a vision towards a free, open, and decentralized world computer."
       },
       {
         "id": 2,
         "imageUrl": "https://i.imgur.com/9BQhqPY.png",
         "title": "Building blocks of Polkadot",
-        "details": "Think of Polkadot like building with Legos - you can pick and choose the pieces you want to create something unique! Imagine building a hyper-optimized blockchain, called a parachain, with specific characteristics to fit your exact need! All this is possible with Polkadot!"
+        "details": "Polkadot is designed to solve 3 fundamental problems in the blockchain space: scalability, security, and interoperability. The Polkadot ecosystem is composed of many independent web3 applications, like blockchains and DApps. Imagine building or using hyper-optimized blockchains, called a parachains, designed to fit to your exact needs! All of this is possible with Polkadot!"
```

Comment:
maybe something about contracts as well? 

Also didn't click with the phrase "hyper-optimized blockchains", but that might be subjective.